cmd/local-admin-client.go

```diff
@@ -97,6 +98,37 @@ func (lc localAdminClient) WriteTmpConfig(tmpFileName string, configBytes []byte
 	return err
 }
 
+// BackupConfig - saves a copy of the current config file
+func (lc localAdminClient) BackupConfig() (retErr error) {
+	configFile := getConfigFile()
```

Comment:
Is it possible for the context to be passed to this function?

Reply:
> Is it possible for the context to be passed to this function?

getConfigFile() ? it is not returning any error, we don't have to pass any function here

Reply:
No, I meant `BackupConfig`

Reply:
> No, I meant BackupConfig

I am not sure how this could be useful. In each method of localAdminClient, we already create a new context and use it

Reply:
> I am not sure how this could be useful. In each method of localAdminClient, we already create a new context and use it

oh I think I understood, you want a context to be created in a more top level and avoid creating a new context in each method of localAdminClient ?

Reply:
Yes, at the handler level